app/components/PaymentModal.tsx

```diff
@@ -0,0 +1,189 @@
+    </>
+  );
+};
+
+const PaymentForm: FC<{
+  handleSendPayment: (event: React.FormEvent<HTMLFormElement>) => void;
+  handleInputChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}> = ({ handleSendPayment, handleInputChange }) => {
+  return (
+    <div>
+      <form action="" className="flex flex-col" onSubmit={handleSendPayment}>
+        <label htmlFor="amount" data-cy="amount-label">
+          Amount
+        </label>
+        <input
+          className="px-2 py-1 placeholder-blueGray-300 text-black relative bg-white rounded text-sm border-0 shadow outline-none focus:outline-none focus:ring w-5/6"
+          type="text"
+          name="amount"
+          id="amount"
+          onChange={handleInputChange}
+          data-cy="amount-input"
+        />
+        <label
+          htmlFor="destinationPublicKey"
+          data-cy="destination-account-label"
+        >
+          Destination
+        </label>
+        <input
+          className="px-2 py-1 placeholder-blueGray-300 text-black relative bg-white rounded text-sm border-0 shadow outline-none focus:outline-none focus:ring w-5/6"
+          type="text"
+          name="destinationPublicKey"
+          id="destinationPublicKey"
+          onChange={handleInputChange}
+          data-cy="destination-account-input"
+        />
+        <label htmlFor="fee" data-cy="fee-label">
+          Fee
+        </label>
+        <input
+          className="px-2 py-1 placeholder-blueGray-300 text-black relative bg-white rounded text-sm border-0 shadow outline-none focus:outline-none focus:ring w-5/6"
+          type="text"
+          name="fee"
+          id="fee"
+          onChange={handleInputChange}
+          data-cy="fee-input"
+        />
+        <label htmlFor="timeOutInSeconds" data-cy="time-out-label">
+          Timeout
+        </label>
+        <input
+          className="px-2 py-1 placeholder-blueGray-300 text-black relative bg-white rounded text-sm border-0 shadow outline-none focus:outline-none focus:ring w-5/6"
+          type="text"
+          name="timeOutInSeconds"
+          id="timeOutInSeconds"
+          onChange={handleInputChange}
+          data-cy="time-out-input"
+        />
+        <label htmlFor="memo" data-cy="memo-label">
+          Memo
+        </label>
+        <input
+          className="px-2 py-1 placeholder-blueGray-300 text-black relative bg-white rounded text-sm border-0 shadow outline-none focus:outline-none focus:ring w-5/6"
+          type="text"
+          name="memo"
+          id="memo"
+          onChange={handleInputChange}
+          data-cy="memo-input"
+        />
+        <label htmlFor="signerKey" data-cy="signer-account-label">
+          Signer
+        </label>
+        <input
+          className="px-2 py-1 placeholder-blueGray-300 text-black relative bg-white rounded text-sm border-0 shadow outline-none focus:outline-none focus:ring w-5/6"
+          type="password"
+          name="signerKey"
+          id="signerKey"
```

Comment:
Ids must be written in `kebab-case`.

Reply:
I have changed these IDs to be in kebab-case format.